doc/classes/CollisionShape3D.xml

```diff
@@ -29,6 +29,12 @@
 		</method>
 	</methods>
 	<members>
+		<member name="debug_color" type="Color" setter="set_debug_color" getter="get_debug_color" default="Color(0, 0, 0, 0)">
+			The collision shape color. If this is reset to its default value of [code]Color(0, 0, 0, 0)[/code], the value of [member ProjectSettings.debug/shapes/collision/shape_color] will be used instead.
+		</member>
+		<member name="debug_fill" type="bool" setter="set_enable_debug_fill" getter="get_enable_debug_fill" default="false">
+			Show a solid preview collision in addition to the wireframe.
```

Comment:
```suggestion
			If [code]true[/code] displays a solid preview in addition to the wireframe.
```
Or
```suggestion
			If [code]true[/code], when the shape is displayed, it will also show a solid color in addition to its wireframe.
```